### PR TITLE
Chore/remove dependency on shared functions

### LINKED ISSUE #617: remove dependency on shared notifications

### PR DESCRIPTION
**Pull request by squigglybob**
resolves #617

### CODE REVIEW

File: site/enqueue.php [hunk 1]
```diff
@@ -6,16 +6,23 @@ function zume_training_magic_url_base_allowed_js( $allowed_js = [] ) {
     $allowed_js[] = 'jquery';
     $allowed_js[] = 'jquery-core';
     $allowed_js[] = 'jquery-migrate';
-    $allowed_js[] = 'lodash';
-    $allowed_js[] = 'lodash-core';
     $allowed_js[] = 'moment';
```
Comment (corsacca) [inline site/enqueue.php:9]: @squigglybob,
Just fyi moment is old an no longer maintained. We're slowly removing it from D.T

**Comment by corsacca** [comment]
Nice Work @squigglybob!

**Comment by squigglybob** [comment]
Still going. Need to test everything I touched to make sure it's not broken

**Comment by squigglybob** [comment]
Yes, we are also using luxon primarily for time and not moment, so I may be
able to extract this one out as well

File: site/enqueue.php (hunk 1 above)
Comment (squigglybob) [reply site/enqueue.php:9]: 👍🏻 great thanks. We weren't really using it, so  i've pulled that as well

**Pull request merged**